engine-rest/engine-rest-openapi/src/main/templates/models/org/camunda/bpm/engine/rest/dto/telemetry/TelemetryDataDto.ftl

```diff
@@ -0,0 +1,20 @@
+<#macro dto_macro docsUrl="">
+<@lib.dto>
+
+    <@lib.property
+        name = "installation"
+        type = "string"
+        desc = "An id that is stored as process engine configuration property."/>
```

Comment:
We should have the same description as in the REST API docs here, i.e. with the clarification that this is unique per database.